src/utils/episode-code.ts

```diff
@@ -0,0 +1,15 @@
+export function episodeCode(epSeason: number, epNum: number): string {
```

Comment:
Good planning for having this as a standalone function rather than manually integrating the "S" and "E" parts of the code